components/Hero.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Button } from "@/components/ui/button";
+import NavbarAnima from "@/components/header/NavbarAnima"
+import { AnimatedTitle } from "@/components/AnimatedTitle";
+
+interface HeroProps {
+  onViewEventsClick: () => void;
+}
+
+export const Hero: React.FC<HeroProps> = ({ onViewEventsClick }) => {
+  return (
+    <section className="w-full h-[810px] bg-[url(https://c.animaapp.com/VOhWj8ET/img/adobestock-443655134-1.png)] bg-cover bg-center relative">
```

Comment:
This needs to be our image. TODO: go through and change the anima hosted images to our own images